.github/workflows/release_github.yml

```diff
@@ -4,33 +4,79 @@ on:
   workflow_dispatch:
     inputs:
       version_level:
-        description: "Semantic version level increase."
+        description: Semantic version level increase
         required: true
         type: choice
+        default: patch
         options:
           - patch
           - minor
           - major
-
+          
 permissions:
   contents: write
-  pull-requests: write
   packages: read
+  statuses: read
+  checks: read
+  pull-requests: write
+  actions: read
+  repository-projects: read
+
+env:
+  GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  GH_PAGER: cat
 
 jobs:
+  checks:
```

Comment:
the check for github token validity got lost in the changes. See [my previous comment](https://github.com/EIT-ALIVE/eitprocessing/pull/338#discussion_r1888374719) for what that should look like

Reply:
No, I took this out on purpose.

The token validity check does not work with `secrets.GITHUB_TOKEN`, because it's generated for the specific run and will not validate an HTTP request. Also, this token is *always valid* for the same reason. 

`secrets.GH_PAT` is no longer used.